pages/bellapps.md

```diff
@@ -35,12 +35,13 @@ You will be shown the page below. Make sure you fill it out completely.
 ![Become an Administrator](uploads/images/become_admin.png)
 
 Next, you will need to fill out your configurations. Make sure you write your name all in lowercase letters and your code all in uppercase letters, as in the example below:
```

Comment:
@Supriyaranjan No, in fact I would get rid of the word `note`. We seem to use a lot of extra words in the first steps. See my other comment for what I think looks good.